### PR TITLE
Notification enhanced

### PR DESCRIPTION
**Pull request by lorenzoPrimi**
Fixes https://github.com/ooni/probe/issues/1242

### CODE REVIEW

File: ooniprobe.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1669,7 +1669,7 @@
 				IBC_WARNINGS = NO;
 				IBSC_WARNINGS = NO;
 				INFOPLIST_FILE = ooniprobe/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```
Comment (lorenzoPrimi) [inline ooniprobe.xcodeproj/project.pbxproj:1672]: iOS 10 is the min version to use UNUserNotificationCenterDelegate

File: ooniprobe/AppDelegate.mm [hunk 2]
```diff
@@ -87,6 +87,18 @@ -(void)application:(UIApplication *)application didReceiveLocalNotification:(UIL
     [[NSNotificationCenter defaultCenter] postNotificationName:@"networkTestEnded" object:nil];
 }
 
+//Called when you tap on a notification
+- (void)userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)(void))completionHandler
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.mm:91]: can you wrap this line, please?

File: ooniprobe/AppDelegate.mm [hunk 3]
```diff
@@ -87,6 +87,18 @@ -(void)application:(UIApplication *)application didReceiveLocalNotification:(UIL
...
+{
+    NSDictionary* countlyPayload = response.notification.request.content.userInfo[@"c"];
+    NSLog(@"COUNTLYLOG didReceiveNotificationResponse %@", response.notification.request.content.userInfo);
```
Comment (bassosimone) [inline ooniprobe/AppDelegate.mm:94]: can you remove this log line, please?

**Review by bassosimone — CHANGES_REQUESTED** [review]
Just two cosmetic fixes to go!

**Review by bassosimone — APPROVED** [review]
🐳

**Pull request merged**